tests/testthat/test-model_parameters.aov_es_ci.R

```diff
@@ -1,614 +1,436 @@
-if (require("insight") && require("effectsize") && require("testthat") && require("lme4") && require("parameters")) {
```

Comment:
why have these been removed?

Reply:
I am not sure what these tests add anything over and above the stricter tests. The stricter tests covers all aspects of the returned output.

Reply:
Ok, got it. But maybe you should not skip them everywhere? It should at least run on GitHub-CI.